sketch_map_tool/oqt_analyses/oqt_client.py

```diff
@@ -43,5 +43,10 @@ def get_report(bbox: Bbox, include_svg: bool = True, include_html: bool = False)
             )
         else:
             raise OQTReportError(req.json()["detail"])
-    report_properties = req.json()["properties"]
+    try:
+        report_properties = req.json()["properties"]
+    except requests.exceptions.InvalidJSONError:
+        raise OQTReportError(
+            "There seems to be a problem with OQT. Please try again later."
+        )
```

Comment:
Probably we should first execute `req.raise_for_status()`:
> If we made a bad request (a 4XX client error or 5XX server error response), we can raise it with `raise_for_status()`.

I suspect that this will be enough to cover this case (and also more cases), since a 200 status code and an invalid JSON should not be returned by OQT.

References:
- https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content
- https://requests.readthedocs.io/en/latest/user/quickstart/#response-status-codes

Reply:
Here is an example how we use raise for status in OQT: https://github.com/GIScience/ohsome-quality-analyst/blob/main/workers/ohsome_quality_analyst/ohsome/client.py#L106